packages/flutter/lib/src/material/text_field.dart

```diff
@@ -278,6 +279,11 @@ class TextField extends StatefulWidget {
   /// [Decoration.enabled] property.
   final bool enabled;
 
+  /// When the keyboard pops up and this [EditableText] is in a scrollable container it will attempt to
```

Comment:
This doc comment is not 100% accurate, for example: When you tap on an unfocused Textfield, that's half-way hidden by the leading (e.g. top) edge it will also scroll into view and be aligned scrollPadding pixels below the leading edge. The doc comment should probably be more general and describe the keyboard case as an example where the behavior is most often observed.